eapp/app.js

```diff
@@ -135,13 +135,7 @@ module.exports = () => {
     next();
   });
 
-  app.use(express.static(path.join(__dirname, 'public/css')))
-  app.use(express.static(path.join(__dirname, 'public/html')))
-  app.use(express.static(path.join(__dirname, 'public/js')))
-  app.use(express.static(path.join(__dirname, 'public/terms')))
-  app.use(express.static(path.join(__dirname, 'public/lib')))
-  app.use(express.static(path.join(__dirname, 'public/images')))
-
+  app.use(express.static(path.join(__dirname, 'public/')))
```

Comment:
is `public` necessary or could we use something like: `app`

Reply:
No. It is in the general structure of an express app where static resources are put in public directory.

Reply:
and this cannot be changed i presume?

could we change the route for each static app through express, so we avoid `/public`?

Reply:
@satra Sorry, for the confusion. Please see my updated detailed answer. 
Also note that /public will not be there in the routes. For example, if you are refereing to a nda module html file, then you can access by 127.0.0.1/nda/html/x.html

Reply:
Reorganized the files to based on S2 structure (as mentioned above).